VotingApplication/VotingApplication.Web/Api/Controllers/ManageMiscController.cs

```diff
@@ -36,6 +36,7 @@ public void Put(Guid manageId, ManagePollMiscRequest updateRequest)
                 poll.InviteOnly = updateRequest.InviteOnly;
                 poll.NamedVoting = updateRequest.NamedVoting;
                 poll.OptionAdding = updateRequest.OptionAdding;
+                poll.HiddenResults = updateRequest.HiddenResults;
```

Comment:
Minor, but we should make sure we test that the updateRequest is applied to the poll properly